src/vm/wren_core.wren

```diff
@@ -302,6 +302,52 @@ class List is Sequence {
     }
     return result
   }
+
+  sort() { sort_(this.map {|n| [n, n] }.toList) }
+
+  sort(key) { sort_(this.map {|n| [n, key.call(n)] }.toList) }
```

Comment:
Maybe emulate Ruby's `sort`, `sort!` and `sort_by` API: https://ruby-doc.org/core-2.2.0/Array.html#method-i-sort. The current `sort(key)` would map to `sort_by`. Ruby's `sort!` woud map to `sort` and Ruby's `sort` would be `sorted`.